design/media_types.go

```diff
@@ -90,15 +90,18 @@ var workItemType = MediaType("application/vnd.workitemtype+json", func() {
 	Attribute("version", Integer, "Version for optimistic concurrency control")
 	Attribute("name", String, "User Readable Name of this item type")
 	Attribute("fields", HashOf(String, fieldDefinition), "Definitions of fields in this work item type")
+	Attribute("parentPath", String, "The names of the parent types separated with '/' or just '/' if no parents")
 
 	Required("version")
 	Required("name")
 	Required("fields")
+	Required("parentPath")
 
 	View("default", func() {
 		Attribute("version")
 		Attribute("name")
 		Attribute("fields")
+		Attribute("parentPath")
```

Comment:
I was in doubt about this as well. Not sure at all if it's a good idea to expose the parent path. But if we don't need it for clients why do we need it in the DB? I think some alignment here would be a good think.
But as I said before when updating the existing WIPs during migration I can workaround this inconsistency between model and DB working with DB parent-path fields directly and not using model.WIP if we really don't want to expose parentPath.